docs/analytics/api.mdx

```diff
@@ -4,11 +4,11 @@ The Sourcegraph Analytics API is an API that provides programmatic access to you
 
 ## Access tokens
 
-For Sourcegraph Analytics, you can generate an access token for programmatic access. Tokens are long-lived with an optional expiry and have the same permissions to access instance data as the user who created them.
+For Sourcegraph Analytics, you can generate an access token for programmatic access using your session cookie. Tokens are long-lived with an optional expiry and have the same permissions to access instance data as the user who created them.
 
-### Token management APIs
+### Getting Started
 
-Token management is currently only available via the Sourcegraph Analytics API. Token management APIs are authenticated via the `cas` session cookie.
+Tokens are first retrieved via the `cas` session cookie and then used to generate a long-lived access token. Token management is currently only available via the Sourcegraph Analytics API.
```

Comment:
> Tokens are first retrieved via the `cas` session cookie

> and then used to generate a long-lived access token

This is worded as if there are two different tokens
1. retrieved via cas cookie
2. long lived token generated using 1.

Reply:
how about this?

"Access tokens are generated using your `cas` session cookie to create long-lived tokens for programmatic access. Token management is currently only available via the Sourcegraph Analytics API."

Reply:
```suggestion
Access tokens are created using the `cas` cookie for authentication. Access tokens are longer lived than the `cas` cookie making them more suitable for programmatic access to the Sourcegraph Analytics API.
```